app/src/app.config.json

```diff
@@ -15,7 +15,7 @@
     "folderRules": true,
     "tagsEnabled": true,
     "categoriesEnabled": true,
-    "knowledgeRetrievalEnabled": false
+    "knowledgeRetrievalEnabled": true
```

Comment:
Should KR be enabled by default?

Reply:
sorry, I pushed this by accident. I enabled it for local testing. I reverted that change now as it should be disabled